doc/design/embargo-redesign.md

```diff
@@ -116,3 +116,107 @@ sequenceDiagram
+
+**Remediation.** None needed, despite the apparent contradiction. Though Dandiset 2 contains a world-readable asset, no one who is not authorized to view the contents of Dandiset 2 knows that fact; for all they know, any data at all, *including open assets stored in the archive*, could be in that Dandiset. Thus, no excess information about Dandiset 2 is leaked.
+
+### Open Dandiset includes embargoed asset
+
+**Scenario.** Roughly the opposite of the last scenario: someone creates an *embargoed* Dandiset 3, uploading asset B to it. They then create an *open* Dandiset 4, also uploading B to that.
+
+**Quirk.** Dandiset 4 now contains a hidden asset, which contradicts the openness of Dandiset 4 and seems to compromise the secrecy of Dandiset 3.
+
+**Remediation.** There are a couple of ideas here:
+
+1. **Don’t allow embargoed Dandiset assets to be used in open Dandisets.** At upload time, the API would simply refuse to allow hidden assets that are part of any embargoed Dandiset to be uploaded to an open Dandiset.
+
+   This cuts the problem off at the root, but it is subject to a failure of our imaginations: perhaps there is a valid reason a neuroscientist would want to do this. One example: if a standard community tool exists that generates a skeleton Dandiset that includes something boilerplateish like a license file or copyright notice, people would regularly include those in embargoed Dandisets, and that could trigger this error condition somewhat baselessly.
+
+   On the other hand, perhaps scientists would have a reason to want to include the same data in both an embargoed Dandiset and an open one. For instance, perhaps embargoed experiments are using some base but non-secret data within an embargoed Dandiset, and that same data needs to be part of an open Dandiset.
+
+2. **Allow this type of sharing but force the user to provide an explicit flag to do so.** In this remediation, the user would receive a warning that they are trying to include embargoed data in an open Dandiset. The API / client would return an error with a prompt to retry with a `--force` type flag, which would then allow the upload to occur. The embargoed assets would remain hidden to the public in the S3 bucket, and data access to the open Dandiset’s asset holding would occur via presigned URLs (the same way as for authorized users to embargoed Dandiset assets).
+
+The major questions here are:
+
+- **Are the semantics valid of placing an embargoed asset in an open Dandiset?**
+  *Answer:* Yes. Doing so causes the asset to become unhidden (see "chosen
+	solution" section below).
+
+- **Would a scientist ever want to push data to an embargoed Dandiset, and also
+	to an open Dandiset?** *Answer:* Maybe, but it doesn't matter--if they do
+	this, they'll be warned about their asset within their embargoed Dandiset
+	becoming open, and they can act according to their own wishes.
```

Comment:
yup, this is correct and reflects the above. i don't know if we want to capture the full discussion. if yes, then ignore my above comments.